server/database.go

```diff
@@ -701,21 +701,42 @@ func (s *Server) composeDatabaseRelationship(ctx context.Context, database *api.
 	}
 
 	rowStatus = api.Normal
-	labels, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
+	labelList, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
 		DatabaseID: &database.ID,
 		RowStatus:  &rowStatus,
 	})
 	if err != nil {
 		return err
 	}
 
-	if len(labels) > 0 {
-		labels, err := json.Marshal(labels)
-		if err != nil {
-			return err
-		}
-		database.Labels = string(labels)
+	// 	Since tenants are identified by labels in deployment config, we need an environment label
+	// to identify tenants from different environment in a schema update deployment.
+	// If we expose the environment label concept in the deployment config, we need it consistently in the label API.
```

Comment:
If we expose the environment label concept in the deployment config, it will look consistent in the label API.

Reply:
> If we expose the environment label concept in the deployment config, it will look consistent in the label API.

I think "it should look consistent in the label API" might be better?

Reply:
Works for me